inventory/environ.py

```diff
@@ -59,6 +59,7 @@
 reVarPrefix = r"${varPrefix}(.*)"
 envPrefix = "SPLUNK_ROLE_"
 reNamePattern = r"${envPrefix}(.*)"
+udsSocketFilename = "/opt/splunkforwarder/var/run/splunk/cli.socket"
```

Comment:
Would be better if we can use the SPLUNK_HOME env parameter for the /opt/splunkforwarder part of this path

Reply:
removed here: https://github.com/splunk/splunk-ansible/pull/843